client/src/components/GameInvitation.js

```diff
@@ -0,0 +1,64 @@
+/*
+Component for inviting friends to a new game.
+*/
+
+import React, { useState, useEffect } from "react";
+import { Grid, Button } from '@material-ui/core';
+import LinkIcon from '@material-ui/icons/Link';
+
+function GameInvitation() {
+
+    // States for user input and list of emails invited.
+    const [emailInput, setEmailInput] = useState("");
+    const [emailList, setEmailList] = useState([]);
+
+    // Updates the state when the user input changes.
+    const onInputChange = (e) => {
+        setEmailInput(e.target.value);
+    }
+
+    // Add the input to the list of emails then clear the input.
+    // Need to implement actual sending of the email.
+    const sendInvitation = () => {
+
+        // Only if the user input is not empty.
+        if (emailInput.length > 0) {
```

Comment:
The instinct to document your code is good, but you've written your code pretty clearly here so these comments are not necessary to understand the functionality. It's good to keep comments minimal because they can pile up over time and really impact readability